<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(171, 177, 255);</string>
    </property>
    <property name="text">
     <string>监控界面</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(63, 63, 63);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>641</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(63, 63, 63)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 218, 218);</string>
    </property>
    <property name="text">
     <string>工作模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 218, 218);</string>
    </property>
    <property name="text">
     <string>温度范围</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelModel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 218, 218);</string>
    </property>
    <property name="text">
     <string>——</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemRange">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 218, 218);</string>
    </property>
    <property name="text">
     <string>——</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234,234,234);</string>
    </property>
    <property name="title">
     <string>从机一</string>
    </property>
    <widget class="QLabel" name="labelRoomNum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>房间号</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelModel_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>311</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelState">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateLabel_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSpeed">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>风速</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>当前温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>52</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>目标温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel_1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="curTem_1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="tarTem_1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>55</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="showIDLabel_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTextCost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>102</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>费用</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCost_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QToolButton" name="ButtonDetail_1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>显示详单</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234,234,234);</string>
    </property>
    <property name="title">
     <string>从机三</string>
    </property>
    <widget class="QLabel" name="labelRoomNum_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>房间号</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelModel_17">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>313</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelState_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateLabel_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSpeed_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>风速</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>当前温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>51</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>目标温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="curTem_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="tarTem_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>45</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelID_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="showIDLabel_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTextCost_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>费用</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCost_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QToolButton" name="ButtonDetail_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>显示详单</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234,234,234);</string>
    </property>
    <property name="title">
     <string>从机四</string>
    </property>
    <widget class="QLabel" name="labelRoomNum_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>房间号</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelModel_27">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>315</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelState_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateLabel_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSpeed_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>风速</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>当前温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>52</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>目标温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="curTem_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="tarTem_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>45</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelID_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="showIDLabel_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTextCost_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>费用</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCost_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QToolButton" name="ButtonDetail_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>显示详单</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButtonList">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>报表</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonPower">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开机</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>60</y>
      <width>651</width>
      <height>541</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234,234,234);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234,234,234);</string>
    </property>
    <property name="title">
     <string>从机二</string>
    </property>
    <widget class="QLabel" name="labelRoomNum_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>房间号</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelModel_12">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>312</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelState_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="stateLabel_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSpeed_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>风速</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>当前温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemNow_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>52</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>目标温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="curTem_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="tarTem_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>45</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelID_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="showIDLabel_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTextCost_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>费用</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCost_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QToolButton" name="ButtonDetail_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>显示详单</string>
     </property>
    </widget>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>labelModel</zorder>
   <zorder>labelTemRange</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>toolButtonList</zorder>
   <zorder>toolButtonPower</zorder>
   <zorder>label</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
